src/utils/debug.ts

```diff
@@ -1,27 +0,0 @@
-/* eslint-disable no-console */
-
-export const isDev = process.env.NODE_ENV === 'development';
-export const isVerbose = process.env.VERBOSE === 'true';
-
-export const log: any = (
```

Comment:
Just curious, why you decided to replace this with console?

Reply:
Nobody actually used this, other than old legacy components.

I noticed we tend to favor using `console.log` or `console.error` for the most part. I actually don't know if many devs were even aware this existed in our codebase.

So i decided to treat it as unneeded legacy, and remove it.

If you have a strong argument about it being in the codebase, however, feel free to argue it, and I'll put it back.